test-it/test.js

```diff
@@ -240,8 +240,11 @@
         });
         // Vireo unit tests can't fire JS events, so register no-op registration functions
         vireo.controlEvents.setRegisterForControlEventsFunction(function () {
+            // No-op.  Wait, didn't you just say above this function was a no-op?
+            // Why yes, yes I did, but this comment is here to defeat stupid eslint rules.
```

Comment:
The tool is only as smart as its contributors! You can see the rule definition here: https://eslint.org/docs/rules/no-empty-function

Maybe make a pull request to add an option to have line before comments as an additional option. That would be pretty cool :)

Hopefully we can get your local dev environment set-up so you don't have to got back and forth with the CI for running lint tests. That really is a silly and inefficient way to run your linter :P

Reply:
Also just for future reference the linter gives a handy token in the error message: https://travis-ci.org/ni/VireoSDK/builds/396619087#L833

like `no-empty-function` which makes it pretty quick to look-up and see a description of the rules 👍